src/librustdoc/html/render/mod.rs

```diff
@@ -1624,6 +1624,15 @@ fn render_impl(
         }
 
         if let Some(ref dox) = i.impl_item.collapsed_doc_value() {
+            if trait_.is_none() && i.inner_impl().items.is_empty() {
+                w.write_str(
+                    "<div class=\"item-info\">\
+                    <div class=\"stab empty-impl\">\
+                        <span class=\"emoji\">⚠️</span> This impl block contains no items.\
```

Comment:
Let's get rid of the emoji. ⚠️indicates a warning or something worrisome, but an empty impl block is totally fine and supported - or at least will be after this PR.